src/api/cpp/cvc5_kind.h

```diff
@@ -2425,147 +2864,201 @@ enum Kind : int32_t
+   *
+   * - Create Op of this kind with:
+   *   - Solver::mkOp(Kind) const
    */
   BAG_MAP,
   /**
-    * bag.filter operator filters the elements of a bag.
-    * (bag.filter p B) takes a predicate p of type (-> T Bool) as a first
-    * argument, and a bag B of type (Bag T) as a second argument, and returns a
-    * subbag of type (Bag T) that includes all elements of B that satisfy p
-    * with the same multiplicity.
-    *
-    * Parameters:
-    *   - 1: a function of type (-> T Bool)
-    *   - 2: a bag of type (Bag T)
-    *
-    * Create with:
-    *   - `Solver::mkTerm(Kind kind, const Term& child1, const Term& child2) const`
-    *   - `Solver::mkTerm(Kind kind, const std::vector<Term>& children) const`
-    */
+   * Bag filter.
+   *
+   * This operator filters the elements of a bag.
+   * `(bag.filter p B)` takes a predicate @f$p@f$ of Sort
+   * @f$(\rightarrow S_1 \; S_2)@f$ as a first
+   * argument, and a bag @f$@f$ of Sort (Bag @f$S@f$) as a second argument, and
+   * returns a subbag of Sort (Bag @f$T@f$) that includes all elements of
+   * @f$B@f$ that satisfy @f$p@f$ with the same multiplicity.
+   *
+   * - Arity: `2`
+   *   - `1:` Term of function Sort @f$(\rightarrow S_1 \; S_2)@f$
+   *   - `2:` Term of bag Sort (Bag @f$S_1@f$)
+   *
+   * - Create Term of this Kind with:
+   *   - Solver::mkTerm(Kind, const std::vector<Term>&) const
+   *   - Solver::mkTerm(const Op&, const std::vector<Term>&) const
+   *
+   * - Create Op of this kind with:
+   *   - Solver::mkOp(Kind) const
+   */
    BAG_FILTER,
   /**
-   * bag.fold operator combines elements of a bag into a single value.
-   * (bag.fold f t B) folds the elements of bag B starting with term t and using
-   * the combining function f.
+   * Bag fold.
    *
-   * Parameters:
-   *   - 1: a binary operation of type (-> T1 T2 T2)
-   *   - 2: an initial value of type T2
-   *   - 2: a bag of type (Bag T1)
+   * This operator combines elements of a bag into a single value.
+   * `(bag.fold f t B)` folds the elements of bag @f$B@f$ starting with term
+   * @f$t@f$ and using the combining function @f$f@f$.
    *
-   * Create with:
-   *   - `Solver::mkTerm(Kind kind, const Term& child1, const Term& child2, const Term& child3) const`
-   *   - `Solver::mkTerm(Kind kind, const std::vector<Term>& children) const`
+   * - Arity: `2`
+   *   - `1:` Term of function Sort @f$(\rightarrow S_1 \; S_2 \; S_2)@f$
+   *   - `2:` Term of Sort `S_2)` (the initial value)
+   *   - `3:` Term of bag Sort (Bag @f$S_1@f$)
+   *
+   * - Create Term of this Kind with:
+   *   - Solver::mkTerm(Kind, const std::vector<Term>&) const
+   *   - Solver::mkTerm(const Op&, const std::vector<Term>&) const
+   *
+   * - Create Op of this kind with:
+   *   - Solver::mkOp(Kind) const
    */
   BAG_FOLD,
   /**
    * Table cross product.
    *
-   * Parameters:
-   *   - 1..2: Terms of bag sort
+   * - Arity: `2`
+   *   - `1..2:` Terms of bag Sort
```

Comment:
```suggestion
   *   - `1..2:` Terms of table Sort
```

Reply:
We should note that table is a bag of tuples.  This is analogous to relation = set of tuples, which we should handle in the same way in the API documentation.

Reply:
Where would you note that? We don't mention this for bags in the kind documentation afaik.

Reply:
Maybe above the first `TABLE_` and `RELATION_` kinds?

Reply:
I have added documentation to the Sort documentation.